examples/python/pincell_depletion/run_depletion.py

```diff
@@ -0,0 +1,161 @@
+helium.add_element('He', 2.4044e-4)
+helium.depletable = False
+
+zircaloy = openmc.Material(material_id=3, name='Zircaloy 4')
+zircaloy.set_density('g/cm3', 6.55)
+zircaloy.add_element('Sn', 0.014  , 'wo')
+zircaloy.add_element('Fe', 0.00165, 'wo')
+zircaloy.add_element('Cr', 0.001  , 'wo')
+zircaloy.add_element('Zr', 0.98335, 'wo')
+zircaloy.depletable = False
+
+borated_water = openmc.Material(material_id=4, name='Borated water')
+borated_water.set_density('g/cm3', 0.740582)
+borated_water.add_element('B', 4.0e-5)
+borated_water.add_element('H', 5.0e-2)
+borated_water.add_element('O', 2.4e-2)
+borated_water.add_s_alpha_beta('c_H_in_H2O')
+borated_water.depletable = False
+
+###############################################################################
+#                 Exporting to OpenMC geometry.xml file
+###############################################################################
+
+# Instantiate ZCylinder surfaces
+fuel_or = openmc.ZCylinder(surface_id=1, x0=0, y0=0, R=0.39218, name='Fuel OR')
+clad_ir = openmc.ZCylinder(surface_id=2, x0=0, y0=0, R=0.40005, name='Clad IR')
+clad_or = openmc.ZCylinder(surface_id=3, x0=0, y0=0, R=0.45720, name='Clad OR')
+left = openmc.XPlane(surface_id=4, x0=-0.62992, name='left')
+right = openmc.XPlane(surface_id=5, x0=0.62992, name='right')
+bottom = openmc.YPlane(surface_id=6, y0=-0.62992, name='bottom')
+top = openmc.YPlane(surface_id=7, y0=0.62992, name='top')
+
+left.boundary_type = 'reflective'
+right.boundary_type = 'reflective'
+top.boundary_type = 'reflective'
+bottom.boundary_type = 'reflective'
+
+# Instantiate Cells
+fuel = openmc.Cell(cell_id=1, name='cell 1')
+gap = openmc.Cell(cell_id=2, name='cell 2')
+clad = openmc.Cell(cell_id=3, name='cell 3')
+water = openmc.Cell(cell_id=4, name='cell 4')
+
+# Use surface half-spaces to define regions
+fuel.region = -fuel_or
+gap.region = +fuel_or & -clad_ir
+clad.region = +clad_ir & -clad_or
+water.region = +clad_or & +left & -right & +bottom & -top
+
+# Register Materials with Cells
+fuel.fill = uo2
+gap.fill = helium
+clad.fill = zircaloy
+water.fill = borated_water
+
+# Instantiate Universe
+root = openmc.Universe(universe_id=0, name='root universe')
+
+# Register Cells with Universe
+root.add_cells([fuel, gap, clad, water])
+
+# Instantiate a Geometry, register the root Universe, and export to XML
+geometry = openmc.Geometry(root)
+geometry.export_to_xml()
+
+###############################################################################
+#                   Exporting to OpenMC materials.xml file
+###############################################################################
+
+# Compute cell areas
+area = {}
+area[fuel] = np.pi * fuel_or.coefficients['R'] ** 2
+
+# Set materials volume for depletion. Set to an area for 2D simulations
+uo2.volume = area[fuel]
+
+# Instantiate a Materials collection and export to XML
+materials_file = openmc.Materials([uo2, helium, zircaloy, borated_water])
+materials_file.export_to_xml()
```

Comment:
No need to define/export `materials_file` -- the depletion operator handles this automatically